web/v1/snowflake/sql-runner/configs/XX_destroy_standard.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I prefer your naming here - but for the sake of consistency can we rename to `teardown_all.json`?

We should change them all to this in a follow-up though - could you create an issue for this?